### ISSUE TITLE
Feature request: decimal separator

### ISSUE THREAD

**Issue by quenthal** [Blackymas/NSPanel_HA_Blueprint#1161]
Non-urgent suggestion as user configurable and/or localization dependent setting:

Decimal seperator with either point (. which is current default) or comma (,).

Examples: 
Decimal point 20.2
Decimal comma: 20,2

**Comment by edwardtfn** [comment]
Should that be per language or per user setting?

**Comment by quenthal** [comment]
> Should that be per language or per user setting?

Not sure, I'd reckon per user for maximum customization?

**Comment by edwardtfn** [comment]
Yeah, per panel (user?) will be the maximum customization, but one more entry on the busy Blueprint inputs. Not a problem, but can easily get lost with so many inputs there, so if we do this by language it would be easier for the one setting up the panel.

**Comment by quenthal** [comment]
> Yeah, per panel (user?) will be the maximum customization, but one more entry on the busy Blueprint inputs. Not a problem, but can easily get lost with so many inputs there, so if we do this by language it would be easier for the one setting up the panel.

Ah. Now I get what you mean. No need to clutter Blueprint input UI with this? Maybe per language is better, or even opening it for adv. configuration or similar.

**Comment by marcelloceschia** [comment]
If you need help with implementing this feature, I can support and I have a use case in my family.

**Comment by edwardtfn** [comment]
If you wanna help with this it would be great.
In fact, I don't think it will be a small change. Most (but not all) service calls (from the blueprint to ESPHome) are sending numbers as text strings, many times with the units included (so those are really strings), but there are some cases where this conversion is done on ESPHome side.
Create a function to convert it is much easier on ESPHome side than on the Blueprint, as it's easier to implement scripts on ESPHome, however identify when the `.` should be replaced by `,` and when it should be kept intact could be a challenge.
Maybe a new service to send values (as strings) as one parameter and it's units as another parameter would be a simple way to make it easier to solve, but still requires many changes on both the blueprint and ESPHome side and new service calls will require new major version to ensure both sides are aligned on the new services.
I'm a bit busy these days, but probably can come back to this in a week or two. It would be nice if you can help with tests at that moment.
Which languages are you using?

**Comment by marcelloceschia** [comment]
I would do this because one of my relatives uses the display in german. I haven't started figuring out how easy it is and what's the best place for that conversion, but I will have some more time in December and will start then

**Comment by edwardtfn** [comment]
This was implemented in `dev` and will be part of the next release.
